config/formats.js

```diff
@@ -554,6 +554,39 @@ let Formats = [
 		banlist: ['Blacephalon', 'Kartana', 'Komala', 'Kyurem-Black', 'Porygon-Z', 'Silvally', 'Tapu Koko', 'Tapu Lele', 'Aerodactylite', 'King\'s Rock', 'Metagrossite', 'Razor Fang'],
 		restrictedMoves: ['Acupressure', 'Belly Drum', 'Chatter', 'Geomancy', 'Lovely Kiss', 'Shell Smash', 'Shift Gear', 'Spore', 'Thousand Arrows'],
 	},
+	{
+		name: '[Gen 7] Metronome Battle',
+		threads: [
+			`&bullet; <a href="http://spo.ink/shitmons">Metronome Battle</a>`,
+		],
+
+		mod: 'gen7',
+		gameType: 'doubles',
+		forcedLevel: 666,
+		teamLength: {
+			validate: [2, 2],
+			battle: 2,
+		},
+		searchShow: false,
+		ruleset: ['Cancel Mod', 'HP Percentage Mod'],
+		banlist: [
+			'Aegislash', 'Aerodactylite', 'Aggronite', 'Aguav Berry', 'Ampharosite', 'Berry Juice', 'Black Sludge', 'Blastoisinite',
+			'Blazikenite', 'Charizardite X', 'Charizardite Y', 'Cheek Pouch', 'Cursed Body', 'Desolate Land', 'Diancite', 'Doublade',
+			'Enigma Berry', 'Figy Berry', 'Fur Coat', 'Galladite', 'Garchompite', 'Gardevoirite', 'Grassy Surge', 'Gyaradosite', 'Honedge',
+			'Huge Power', 'Iapapa Berry', 'Ice Body', 'Iron Barbs', 'Kitsunoh', 'Latiasite', 'Latiosite', 'Leftovers', 'Lucarionite',
+			'Mago Berry', 'Metagrossite', 'Moody', 'Normalium Z', 'Oran Berry', 'Parental Bond', 'Poison Heal', 'Power Construct',
+			'Pressure', 'Primordial Sea', 'Protean', 'Pure Power', 'Rain Dish', 'Rocky Helmet', 'Rough Skin', 'Salamencite', 'Sand Stream',
+			'Sceptilite', 'Schooling', 'Shedinja + Sturdy', 'Sitrus Berry', 'Snow Warning', 'Stamina', 'Steelixite', 'Swampertite',
+			'Tyranitarite', 'Venusaurite', 'Wiki Berry', 'Wonder Guard',
+		],
+		onValidateSet: function (set) {
+			let template = this.getTemplate(set.species);
+			if (set.name === template.baseSpecies) return [`${template.species} does not have a nickname set.`, `(Edgy nicknames are required)`];
```

Comment:
Can we <em>not</em> have an official stance of promoting edginess?

Reply:
I can remove the edgy bit if promoting edginess is too much.